### ISSUE TITLE
[CUSA08425]: とある魔術の電脳戦機

### ISSUE THREAD

**Issue by V6ser** [SysRay/psOff_compatibility#764]
### Please, describe what you get

In game with broken graphics.
Game has audio, displays the save warning at boot, then proceed to play with broken graphics.

### Game version you are using

01.00

### Current game status

Ingame

### Your emulator version

Dec  8 2024

### Upload your log there

[20241212-224221.000.zip](https://github.com/user-attachments/files/18119593/20241212-224221.000.zip)

### Upload your screenshots there

![image](https://github.com/user-attachments/assets/eea1a7cc-4ecc-401b-817c-8ac1a6b8d8fd)
![image](https://github.com/user-attachments/assets/ad2a7a92-3de9-4cf8-bdc1-9a416eb52a07)

### Yes, I agree

- [X] Yes, I read the rules and did everything according to them

**Comment by V6ser** [comment]
Graphics improvement. Crashes without save data.
build - December 16th
![image](https://github.com/user-attachments/assets/b09750b4-7826-4676-9caa-1e4919c9aba6)

no crash log - [20241217-155413.000.p7d.zip](https://github.com/user-attachments/files/18170425/20241217-155413.000.p7d.zip)

**Comment by MakutaMaster962** [comment]
Retested with `Feb 23 2025`

### Please, describe what you get

Emulator abruptly "dies" upon clicking the Start button.

### Game version you are using

01.00

### Current game status

Nothing

### Upload your log there

[20250224-110147.000.p7d.zip](https://github.com/user-attachments/files/18947312/20250224-110147.000.p7d.zip)

### Upload your screenshots there

No screenshots :(

**Comment by MakutaMaster962** [comment]
Retested with `2025Jun05`

### Please, describe what you get

Game doesn't even boot up. It just says, "Emulator just died abruptly" (see screenshot below).

### Game version you are using

01.00

### Current game status

Nothing

### Upload your log there

[20250609-161736.000.p7d.zip](https://github.com/user-attachments/files/20661384/20250609-161736.000.p7d.zip)

### Upload your screenshots there

![Image](https://github.com/user-attachments/assets/f03df62f-2e0f-4833-9f3a-7bebfba92350)

**Comment by MakutaMaster962** [comment]
> > Retested with `2025Jun05`
> > ### Please, describe what you get
> > Game doesn't even boot up. It just says, "Emulator just died abruptly" (see screenshot below).
> > ### Game version you are using
> > 01.00
> > ### Current game status
> > Nothing
> > ### Upload your log there
> > [20250609-161736.000.p7d.zip](https://github.com/user-attachments/files/20661384/20250609-161736.000.p7d.zip)
> > ### Upload your screenshots there
> > ![Image](https://github.com/user-attachments/assets/f03df62f-2e0f-4833-9f3a-7bebfba92350)
> 
> You uploaded wrong log file, this one is for main process, the problem is in PS4 system process. Please, upload that log too. If you don't know how to pick correct log, then just close emulator completely, remove everything from your `P7Logs` folder and start the game again. Once it crash, close the user interface and archive entire `P7Logs` directory.

I already followed those exact steps, and that was the only log file that was in the `P7Logs` directory. Like I said before, the game doesn't even boot up when I click the start button. Instead, I just get the "Emulator just died abruptly!" message. That's all the info I can give you. Sorry.

**Comment by MakutaMaster962** [comment]
> > I double-triple-checked the `P7Logs` folder, both before & after trying to run the game, & got the same result once again: only 1 log file was in the folder. And the only info in the terminal window (again, both before & after trying to run the game), was what GPU I have installed on my PC right now (see screenshot below).
> > ![Image](https://github.com/user-attachments/assets/b0aa2419-022f-4cc3-a4d2-6e323a4a3514)
> > I seriously don't know what else I can do.
> 
> How your psOff root directory looks like? Your antivirus removed some vital emulator files as it seems.

I already turned my antivirus completely off, reinstalled the entire emulator, & still got the same result yet again.

My root directory looks like this:

![Image](https://github.com/user-attachments/assets/45541af7-e56e-44ac-9947-638ed106ac39)

The info in the terminal screenshot remains the same before & after I try and run the game. 

Finally, does the fact that my OS is Windows 10 help at all?

**Comment by igor725** [comment]
We support Windows 10, it doesn't really matter. The problem is hidden somewhere else. Can you open this `Problem Reports` window? You should look for psOff_tunnel.exe errors there.

![Image](https://github.com/user-attachments/assets/0c62226f-8ed7-4316-b73d-35a95718d515)

**Comment by igor725** [comment]
> I looked at the `Problem Reports` window, & there was nothing in there for PsOff or PsOff_tunnel.exe. And whenever I try double-clicking on PsOff_tunnel.exe, I get the following error:
> 
> ![Image](https://github.com/user-attachments/assets/b8a51feb-d933-45bd-b71b-bbf73c795bd4)

What's your Windows version? Press Win+R and type `winver` then press Enter

**Comment by igor725** [comment]
Apparently your Windows 10 build is too old and miss some functionality. I will make a workaround for it in next emulator build.

**Comment by igor725** [comment]
> > > I looked at the `Problem Reports` window, & there was nothing in there for PsOff or PsOff_tunnel.exe. And whenever I try double-clicking on PsOff_tunnel.exe, I get the following error:
> > > ![Image](https://github.com/user-attachments/assets/b8a51feb-d933-45bd-b71b-bbf73c795bd4)
> > 
> > 
> > What's your Windows version? Press Win+R and type `winver` then press Enter
> 
> My Windows version is:
> 
> Windows 10 Enterprise LTSC, version: 1809, OS Build: 17763.316

Yes, this version is too old and has no support for `QueueUserAPC2` functionality, the next emulator release will workaround this.

**Comment by MakutaMaster962** [comment]
> > > > I looked at the `Problem Reports` window, & there was nothing in there for PsOff or PsOff_tunnel.exe. And whenever I try double-clicking on PsOff_tunnel.exe, I get the following error:
> > > > ![Image](https://github.com/user-attachments/assets/b8a51feb-d933-45bd-b71b-bbf73c795bd4)
> > > 
> > > 
> > > What's your Windows version? Press Win+R and type `winver` then press Enter
> > 
> > 
> > My Windows version is:
> > Windows 10 Enterprise LTSC, version: 1809, OS Build: 17763.316
> 
> Yes, this version is too old and has no support for `QueueUserAPC2` functionality, the next emulator release will workaround this.

Okay, after a few test runs with the new release, I'm now able to provide the proper log file. In addition to that, I'm also now getting actual problem reports, due to the emulator crashing every time during the game's Training Mode right after the "Versus" screen. Mission Mode has the same results, while Story Mode causes it to crash just when the first "character interaction" dialogue 
screen is supposed to appear.

### Upload your log there

[20250618-195528.000.p7d.zip](https://github.com/user-attachments/files/20808070/20250618-195528.000.p7d.zip)

### Upload your screenshots there

The problem report:
![Image](https://github.com/user-attachments/assets/42ef6068-0697-4300-97e7-5ae9bf940a2b)

The terminal (during crash):
![Image](https://github.com/user-attachments/assets/7d495fab-5554-4205-b9b8-3c0564996955)
